app/controllers/application_controller.rb

```diff
@@ -5,6 +5,15 @@ class ApplicationController < ActionController::Base
 
   protect_from_forgery with: :exception
 
+  def education_ab_test
```

Comment:
@alecgibson @carvil ah I see, yeah that would make sense! I have added a commit to remove the AB test code. Let me know if it seems ok and I will crush it back into the first commit and reword it.